lib/msal-common/src/client/AuthorizationCodeClient.ts

```diff
@@ -228,6 +230,20 @@ export class AuthorizationCodeClient extends BaseClient {
             parameterBuilder.addClaims(request.claims, this.config.authOptions.clientCapabilities);
         }
 
+        if (this.config.systemOptions.preventCorsPreflight) {
```

Comment:
Can we add a comment here explaining that these params will be added to the headers if not provided here?

Reply:
done